package.json

```diff
@@ -10,7 +10,8 @@
     "lintspaces-cli": "0.6.0",
     "stylelint": "9.1.1",
     "stylelint-config-standard": "18.2.0",
-    "webpack": "3.11.0"
+    "webpack": "4.2.0",
```

Comment:
@Crphang Any reason why version `4.2`?

When I did `npm install webpack` , it installed the 4.3.0 version

Reply:
The docs currently is for 4.2 on Webpack official docs. I thought its good to follow that. Later versions of 4.x shouldnt have breaking changes, and can be upgraded iteratively. What do you guys think?

Reply:
Keeping 4.2 sounds fine to me in that case :+1

Reply:
> The docs currently is for 4.2 on Webpack official docs.

Are you sure? I just checked and it's already updated to 4.3.0 which was released only yesterday. I believe it's safe to use the latest version.
Clearly we can't be up-to-date all the time unless we use a non-exact versioning policy, but this shall be a discussion for another day.

Reply:
My folly, didnt check the most recent since my change was super recent as well. Will update it!

Thanks @tshradheya for the windows verification